### DISCUSSION TITLE
Should we remove the `Navbar` after the user signs in?

### DISCUSSION THREAD

**Discussion by dinaolmelak** [Eco-Friends-Connect/Eco-Friends#152] [Q&A]
Websites normally don't have the same nav bar after signing in or signing up. So do we still need to keep the same nav bar after signing in or signing up?

@Donnyd88 @RuthAnn1 @Samueldnvn @Kendiya242

**Comment by Donnyd88** [comment]
I think navbars are the standard. Github as an example has one on this page. I think the user would lose functionality if we removed the navbar. Sometimes those navbars are expressed as burger menus', but just about every site I use frequently has them.  Some examples would be Youtube, my school's website (https://www.unh.edu/), https://x.com/home, popular news sites( https://www.cnn.com/ , https://www.theguardian.com/us), etc.

**Reply by dinaolmelak** [reply]
Thank you @Donnyd88, Sorry for not stating my question more clearly. I don't mean to remove the nav bars completely. I meant after the user signs in that we could maybe change the links. maybe the change would look like this for the nav links:
### From 
| [Home](https://eco-friends-web.web.app/home) | [About](https://eco-friends-web.web.app/about) | [Volunteer Search](https://eco-friends-web.web.app/volunteersearch) | [About The Creators](https://eco-friends-web.web.app/aboutthecreators) |
| --------- | -------- | ------- | ------- |

### To (Organization)
| Dashboard | [About](https://eco-friends-web.web.app/about) | [About The Creators](https://eco-friends-web.web.app/aboutthecreators) |
| --------- | ------- | ------- |

### To (volunteer)
| Userpage | [About](https://eco-friends-web.web.app/about) | [Volunteer Search](https://eco-friends-web.web.app/volunteersearch) | [About The Creators](https://eco-friends-web.web.app/aboutthecreators) |
| --------- | -------- | ------- | ------- |

- My reasoning behind this change is that when the organization or the volunteer logs in, we want them to have user functionalities. 
- After logging in if click out to a different page, there is no way to get back to the previous page without clicking back.
- Also if we set it up this way, we could also integrate the page to let the user set profile pictures.
